src/scss/components/_ratings.scss

```diff
@@ -2,24 +2,25 @@ $star-count: 6;
 $star-size-small: 14px;
 $star-size-medium: 28px;
 $star-size-large: 42px;
+$star-spacing: $star-count;
```

Comment:
Does it make sense to link the count to the spacing – if the count was 100, would you want to `+ 100` to the width each time?

Reply:
When the start rating empty was last updated the size was also updated 1px so that would actually need to happen :) 
I've checked it with a star count of 25 on all 3 sizes of ratings
<img width="404" alt="screen shot 2018-06-27 at 12 18 18" src="https://user-images.githubusercontent.com/5295718/41970863-3a6d72f0-7a04-11e8-82bb-839535985af5.png">
<img width="773" alt="screen shot 2018-06-27 at 12 18 21" src="https://user-images.githubusercontent.com/5295718/41970864-3a8424c8-7a04-11e8-9391-fd31e64f4ba6.png">
<img width="1144" alt="screen shot 2018-06-27 at 12 18 25" src="https://user-images.githubusercontent.com/5295718/41970865-3a9bf062-7a04-11e8-88a0-77e3d2ef3d40.png">


 and all is working as expected:

Reply:
@bulutcy that's my bad - I left in the test count. I have just created a pull request to fix this.

Also - re icon colour - the colour update of the rating icon is in f-icon v0.11.0, this is also in a PR. If you wouldn't mind having a look:  https://github.com/justeat/f-icons/pull/20